packages/components/src/button/README.md

```diff
@@ -15,19 +16,19 @@ Buttons let users take actions and make choices with a single click or tap.
 
 Buttons tell users what actions they can take and give them a way to interact with the interface. You’ll find them throughout a UI, particularly in places like:
 
-- Modals
-- Forms
-- Toolbars
+-   Modals
+-   Forms
+-   Toolbars
```

Comment:
Why all lists are not formatted differently?

Reply:
Hi @youknowriad. I am not sure why, but after installing the prettier extension as in [here](https://developer.wordpress.org/block-editor/contributors/document/#editor-config), every time I saved my file the one space that I inserted was converted to two spaces ¬¬'. Is there anything that I should change in my pretter settings?

Reply:
I think it's probably fine because we don't have strong linting checks for markdown. 👍